data_steward/tools/generate_ext_tables.py

```diff
@@ -132,8 +131,7 @@ def convert_to_bq_string(mapping_list):
     """
     bq_insert_list = []
     for hpo_rdr_item in mapping_list:
-        bq_insert_list.append("(\"{hpo_rdr_id}\", \"{src_id}\")".format(
-            hpo_rdr_id=hpo_rdr_item[0], src_id=hpo_rdr_item[1]))
+        bq_insert_list.append(f"(\"{hpo_rdr_item[0]}\", \"{hpo_rdr_item[1]}\")")
```

Comment:
This is optional. I think this is more readable.  Changing that first double quote to a single quote means we don't need to need to escape the double quotes in the string.
```suggestion
        bq_insert_list.append(f'("{hpo_rdr_item[0]}", "{hpo_rdr_item[1]}")')
```